v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -137,6 +137,16 @@ public interface KafkaToBigQueryFlexOptions
 
   void setSchemaRegistryConnectionUrl(String schemaRegistryConnectionUrl);
 
+  @TemplateParameter.Boolean(
+      order = 11,
+      optional = true,
+      description = "Add the Kafka Key to the BQ table.",
+      helpText = "Add the Kafka Key to the BQ table, in a `_key` field. Default is false (Key is ignored).")
+  @Default.Boolean(false)
+  Boolean getUseKafkaKey();
```

Comment:
Can we rename this to `addKafkaKeyToBqTable` or `addKafkaKey` instead of `useKafkaKey`. 

since we are not using it anywhere

Reply:
Do we want to use `persistKafkaKey` or `addKafkaKey`?

Reply:
persist sg to me

Reply:
Done